scripts/release.sh

```diff
@@ -472,6 +472,7 @@ function run_validation_tests() {
 # Parameters: $1..$n - files to add to the release.
 function publish_artifacts() {
   (( ! PUBLISH_RELEASE )) && return
+  awk '!seen[$0]++' ARTIFACTS_TO_PUBLISH > ARTIFACTS_TO_PUBLISH
```

Comment:
can we use `sort | uniq` instead? `awk` is pretty hard to understand and maintain

Reply:
also, I think it should be at line 522

Reply:
I think we can just use uniq. 
I think we should remove the duplicate in early stage as possible because it is creates redundancy and may introduce other unexpected problems in the future